docker.go

```diff
@@ -1364,13 +1364,17 @@ func (p *DockerProvider) ReuseOrCreateContainer(ctx context.Context, req Contain
 		lifecycleHooks:    []ContainerLifecycleHooks{combineContainerHooks(defaultHooks, req.LifecycleHooks)},
 	}
 
-	if p.config.RyukDisabled {
-		// ryuk disabled, ensure the container we are about to reuse is running
-		if c.State != "running" {
-			// found container to reuse, but it is not in state running
-			if err := dc.Start(ctx); err != nil {
-				return dc, fmt.Errorf("start container %s in state %s: %w", req.Name, c.State, err)
-			}
+	// If a container was stopped programmatically, and Ryuk is enabled, we want to
```

Comment:
question: the comment details Ryuk state but there's no check for this, should there be?

Reply:
Mmm I added it because the original message included it 🤔 But you're right, and readers don't care about Ryuk's state in this case. Removing that asap